tests/integration/examples/test_notebooks_tf2.py

```diff
@@ -0,0 +1,130 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+import pytest
+
+try:
+    import papermill as pm
+    import scrapbook as sb
+except ImportError:
+    pass  # disable error while collecting tests for non-notebook environments
+
+
+# from recommenders.utils.gpu_utils import get_number_gpus
+
+
+TOL = 0.5
+ABS_TOL = 0.05
+
+
+# @pytest.mark.gpu
+# @pytest.mark.integration
+# def test_gpu_vm():
+#     assert get_number_gpus() >= 1
+
+
+@pytest.mark.integration
+@pytest.mark.parametrize(
+    "yaml_file, data_path, epochs, batch_size, expected_values, seed",
+    [
+        (
+            "recommenders/models/deeprec/config/sli_rec.yaml",
+            os.path.join("tests", "resources", "deeprec", "slirec"),
+            5,
+            128,
+            {"ndcg@10": 0.3014, "Hit@10": 0.4875},
+            42,
+        )
+    ],
+)
+#     "size, epochs, expected_values, seed",
+#     [
+#         (
+#             "1m",
+#             5,
+#             {
+#                 "ndcg@10": 0.3014, 
+#                 "Hit@10": 0.4875,
+#             },
+#             42,
+#         ),
+#         # ("10m", 5, {"map": 0.024821, "ndcg": 0.153396, "precision": 0.143046, "recall": 0.056590})# takes too long
+#     ],
+# )
```

Comment:
in the current test pipeline this test will break, we can add:
```
@pytest.mark.skipif(tf.__versoin__> "2.0", reason="We are currently on TF 1.5")
```

Reply:
Added.